Mathlib/Order/InitialSeg.lean

```diff
@@ -27,6 +27,8 @@ These notations belong to the `InitialSeg` locale.
 
 * `r ≼i s`: the type of initial segment embeddings of `r` into `s`.
 * `r ≺i s`: the type of principal segment embeddings of `r` into `s`.
+* `α ≤i β` is an abbreviation for `(· < ·) ≼i (· < ·)`.
+* `α <i β` is an abbreviation for `(· < ·) ≺i (· < ·)`.
```

Comment:
Can you discuss in the module doc why this pair of notations makes sense for initial and principal segments?

Reply:
I just kinda came up with it. My reasoning is that `≼` and `≺` refer to "general" relations, while `≤` and `<` are linked to the order typeclass hierarchy.

Reply:
Sure, but why is "initial segment" the non-strict version of "principal segment"?

Reply:
Because of [`InitialSeg.ltOrEq`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/InitialSeg.html#InitialSeg.ltOrEq). Do I link that lemma in this notation section?

Reply:
I've added a small comment referring to this theorem.